Backend/dev/osrm/README

```diff
@@ -10,4 +10,8 @@ Usage:
 
 Generating Docker image
 1. Download the openstreet data file as mentioned above
-2. `docker build . -f Dockerfile.osrm  -t osrm-custom`
\ No newline at end of file
+2. `docker build . -f Dockerfile.osrm  -t osrm-custom`
+
+Running the Image
+1. docker run osrm-custom
+2. docker cp <container-id>:/data <path-to-local-directory> this will copy /data folder from container to local machine
```

Comment:
why do we need to copy `/data` folder from container to local?

Reply:
for normal purposes we dont need. but for deployments purpose we need to make zip of /data folder which is inside container. should i remove this point or make a section for deployment specific?

Reply:
Why do we need zip of /data folder for deployment?